MWPW-142826

Blainegunn opened the pull request:
Resolves : [MWPW-142826]( https://jira.corp.adobe.com/browse/MWPW-142826)

**Before:**
https://main--dc--adobecom.hlx.page/acrobat/business/enterprise
https://main--dc--adobecom.hlx.page/acrobat/business/enterprise?akamaiLocale=de
https://main--dc--adobecom.hlx.page/acrobat/features/create-pdf
https://main--dc--adobecom.hlx.page/acrobat/features/create-pdf?akamaiLocale=in

**After**
Regular 
https://mwpw-142826--dc--adobecom.hlx.page/acrobat/business/enterprise
https://mwpw-142826--dc--adobecom.hlx.page/acrobat/business/enterprise?akamaiLocale=de

https://mwpw-142826--dc--adobecom.hlx.page/acrobat/business/resources/it-tools
https://mwpw-142826--dc--adobecom.hlx.page/acrobat/business/resources/it-tools?akamaiLocale=fr

w/ Frag (i updated w/ the -geo in the phone-number link)
https://mwpw-142826--dc--adobecom.hlx.page/acrobat/features/create-pdf
https://mwpw-142826--dc--adobecom.hlx.page/acrobat/features/create-pdf?akamaiLocale=in

TsayAdobe reviewed (approved):
- If there is no phone number, it still imports the script `geo-phoneNumber.js` and calls the geo2 API.
- If fetching geo2 and placeholder JSON failed, it could cause the code after get skipped. They may need exception handling. 
- Do we need to conside [sessionStorage.getItem('akamai')](https://github.com/adobecom/milo/blob/main/libs/features/georoutingv2/georoutingv2.js#L58)?

peller commented on acrobat/scripts/scripts.js:
```diff
@@ -391,4 +402,7 @@ const { ietf } = getLocale(locales);
+
+  const { default: geoPhoneNumber } = await import('./geo-phoneNumber.js');
+  geoPhoneNumber();
 }());
```
let's make sure to test this with the edgeworker. Since we inline this module, I expect we will need to make a corresponding change to replace this relative import with an absolute path.

peller replied:
can we / should we defer this network activity to give the page a chance to load?  is it critical to page load?

peller replied:
will need to do someting like https://github.com/adobecom/dc/blob/stage/edgeworkers/Acrobat_DC_web_prod/main.js#L131

peller commented on acrobat/scripts/geo-phoneNumber.js:
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable compat/compat */
+/* c8 ignore next 56 */
+const geoTwo = await fetch('https://geo2.adobe.com/json/');
```
can you have await at the top level outside an async block?

Blainegunn commented:
@TsayAdobe @peller 
https://github.com/adobecom/dc/pull/594/files
Addressed some of the feedback and thank you both.
I have a follow up ticket James https://jira.corp.adobe.com/browse/MWPW-143726
Merging to stage for more testing
